public/app/features/storage/k8s/K8SPage.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { useAsync } from 'react-use';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { useStyles2 } from '@grafana/ui';
+import { Page } from 'app/core/components/Page/Page';
+import { useNavModel } from 'app/core/hooks/useNavModel';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+
+interface RouteParams {
+  // path: string;
+}
+
+interface QueryParams {
+  // view: StorageView;
+}
+
+interface Props extends GrafanaRouteComponentProps<RouteParams, QueryParams> {}
+
+export default function K8SPage(props: Props) {
+  const styles = useStyles2(getStyles);
+  const navModel = useNavModel('k8s');
+  const info = useAsync(() => {
+    return getBackendSrv().get('/api/k8s/info');
+  });
+
+  const renderView = () => {
+    if (info.value) {
+      return <pre>{JSON.stringify(info.value, null, 2)}</pre>;
+    }
+
+    return <div className={styles.wrapper}>K8s FTW</div>;
```

Comment:
tested locally, and i got it working eventually. it wasn't clear if this was the default message, a success message, or a failure

Reply:
Updated so we now show this:  
<img width="594" alt="image" src="https://user-images.githubusercontent.com/705951/207209650-669b1b96-e523-4fbe-8436-f2bdc347b495.png">

can still use work, but at least it says something is wrong :)